MDbScriptTool/Scripts/app/content-result.js

```diff
@@ -61,11 +61,13 @@
         if (instance && instance.$result && db && db.id) {
             var $dbTable = $('#r' + db.id, instance.$result);
             if ($dbTable.length === 0) {
-                instance.$result.append(`<div id="r${db.id}" class="result-sets-container" tabindex="0"><div class="result-sets-header">${db.label || db.name}<span class="result-sets-meta"></span></div></div>`);
+                var excelBtn = `<i onclick=app.downloadExcel('${db.id}') data-toggle="tooltip" db="${db.id}" title="Download to excel" class="fa fa-file-excel-o float-right text-white excel-btn" style="cursor: pointer; margin-top: 2px;"></i>`;
```

Comment:
- Don't use word "Excel"
- Missing quote for onclick attribute.
- Don't use excel icon. Looks like a close button. Use download icon